src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -51,8 +51,12 @@ export default App.extend({
     const storedState = store.get(`${ this.worklistId }_${ currentUser.id }-v2`);
     const filters = this.getState('filters');
 
-    // NOTE: Allows for new defaults to get added to stored filters
-    if (storedState) storedState.filters = extend({}, filters, storedState.filters);
+    if (storedState) {
+      // NOTE: Allows for new defaults to get added to stored filters
+      storedState.filters = extend({}, filters, storedState.filters);
+
+      storedState.lastSelectedId = null;
```

Comment:
Clears the `state.lastSelectedId` value whenever the worklist page is initially loaded.